docs/Converters/UKBtoBIDS.md

```diff
@@ -0,0 +1,87 @@
+<!-- markdownlint-disable MD046 -->
+# `ukb-to-bids` – Conversion of the UK Biobank (UKB) to BIDS
+
+!!! quote "Description reproduced from the [UK Biobank webpage](https://www.ukbiobank.ac.uk/)"
+    UK Biobank is a large-scale biomedical database and research resource, containing in-depth genetic and health information from half a million UK participants. The database is regularly augmented with additional data and is globally accessible to approved researchers undertaking vital research into the most common and life-threatening diseases. It is a major contributor to the advancement of modern medicine and treatment and has enabled several scientific discoveries that improve human health
+
+## Dependencies
+
+If you installed the core of Clinica, this converter needs no further dependencies.
```

Comment:
Doesn't it require dcm2niix?

Reply:
not right now since we take only the nifti as inputs.